sycl/test-e2e/Matrix/SG32/joint_matrix_annotated_ptr.cpp

```diff
@@ -5,15 +5,14 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
+// VNNI transform and sub-group size 32 are not supported yet on DG2 by IGC
+// UNSUPPORTED: gpu-intel-dg2
 // REQUIRES: aspect-ext_intel_matrix
 // REQUIRES-INTEL-DRIVER: lin: 27501, win: 101.4943
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-// Currently row major B fails when annotated_ptr is used
-// XFAIL: gpu
```

Comment:
Does this test pass on PVC? I did not see any unexpected pass on PVC for this test...

Reply:
I forgot to test this on PVC. I just tested it now and yes you are right, it fails. 
I'm adding an unsupported directive for pvc on these tests.

Reply:
The test is supported on PVC but it is XFAIL.
So you should keep both:
// UNSUPPORTED: gpu-intel-dg2
and
// XFAIL: gpu
(same goes to the other tests)